### PR TITLE
Support for google-map-directions Waypoints

### PR DESCRIPTION
**Pull request by parondeau**
I wasn't sure if there was any formal process required to submit pull requests or if I need to update docs as well, but I've added support for the waypoints functionality as per: 
https://developers.google.com/maps/documentation/javascript/3.exp/reference#DirectionsWaypoint
and 
https://developers.google.com/maps/documentation/javascript/examples/directions-waypoints

### CODE REVIEW

**Comment by parondeau** [comment]
I signed it!

**Comment by atotic** [comment]
Looks clean to me.

File: google-map-directions.html [hunk 1]
```diff
@@ -94,6 +95,21 @@
...
+       * The maximum allowed waypoints is 8, plus the origin, and destination.
+       * Maps API for Business customers are allowed 23 waypoints,
+       * plus the origin, and destination.
+       * Waypoints are not supported for transit directions. Optional.
+       *
+       * @type google.maps.DirectionsWaypoint
+       */
+       waypoints: {
+         type: Array,
+         value: []
```
Comment (ebidel) [inline google-map-directions.html]: `value: function() { return []; }`

**Comment by ebidel** [comment]
LGTM. One nit.

**Pull request merged**